composer.json

```diff
@@ -10,12 +10,13 @@
         {
             "name": "Ben Gibson",
             "email": "ben.gibson.2011@gmail.com",
-            "homepage": "ben-gibson.co.uk",
+            "homepage": "http://ben-gibson.co.uk",
```

Comment:
@ben-gibson, composer complained as this was invalid. I assume http is correct? Couldn't resolve the address from my machine to check.